tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -199,12 +211,12 @@ private void ExecuteFunction(String Function)
                     {
                         String newini = ini.Substring(0, pos + 9);
                         newini += SelectedRenderer;
-                        newini += ini.Substring(pos + 10, ini.Length - pos - 10);
+                        newini += ini.Substring(pos + 11, ini.Length - pos - 11);
```

Comment:
I don't think padding the renderer number with a space so that it's always 2 characters is a good idea. Maybe a better idea would be to look for the following newline and cut everythng out between `Renderer=` and the newline.

Also, I don't think the second `Substring` argument is necessary in this case. Without the second argument, the substring ends at the end of the original string anyway.

Side note: I'm not familiar with C#.

Reply:
I'll leave the padding as it is since it's working as it is unless someone else wants to change it , plus also not that familiar with C# as well :laughing: 

As for the second argument on the Substring I removed it and it's working properly.